ACHNBrowserUI/ACHNBrowserUI/views/todayDashboard/TodayBirthdaysSection.swift

```diff
@@ -11,16 +11,22 @@ import Backend
 import UI
 
 struct TodayBirthdaysSection: View {
-    @StateObject private var viewModel = VillagersViewModel()
+    @Environment(\.currentDate) private static var currentDate
+    @StateObject private var viewModel = VillagersViewModel(currentDate: Self.currentDate)
 
     var headerText: String {
         viewModel.todayBirthdays.count > 1 ? "Today's Birthdays" : "Today's Birthday"
     }
     
     var body: some View {
         Group {
-            if viewModel.todayBirthdays.isEmpty {
+            if viewModel.isLoading {
                 RowLoadingView()
+            } else if viewModel.todayBirthdays.isEmpty {
+                Text("No Birthday today...")
```

Comment:
I'm not sure about this one... should `Birthday` takes an `s`? I think so... but it doesn't sound great I let it without... now I'm wondering :D

Reply:
🤷‍♂️

Reply:
@ericlewis as a native English speaker, any thoughts?